src/launchpad/actions/expressions.js

```diff
@@ -0,0 +1,31 @@
+const constants = require("../constants");
+
+function evaluateInput(evaluate, input) {
+  return (dispatch) => {
```

Comment:
This is a small detail, but I like how the debugger are uses async functions here with await.